cryptographic_flows.md

```diff
@@ -22,8 +22,36 @@ An asset owner that has not previously interacted with the key server MUST regis
     1. Checks that `user_id` matches that of the authenticated user in the open registration session. If this check fails, the server MUST abort.
     1. Checks that the received ciphertext is of the expected format and length.
     1. [Stores](#server-side-storage) the received ciphertext in a record associated with `user_id`.
+    1. Stores the registration registration request in the [audit log](#audit-logs).
 1. The client and server close the session upon completion of the request.
 
+## Retrieve Audit Logs
+The asset owner can request audit logs from the key server.
+
+Input:
```

Comment:
Can we include an optional date range as a parameter?

Reply:
Could we consider adding severity levels across log entry types? Will provide additional granularity and implicit log filtering on requests. Some common levels: `error`, `warning`, `debug`, `info`, and `notice`.